src/main/java/ru/ivi/opensource/flinkclickhousesink/applied/ClickHouseWriter.java

```diff
@@ -72,7 +72,7 @@ private void buildComponents() {
                 Integer.MAX_VALUE,
                 60L,
                 TimeUnit.SECONDS,
-                new LinkedBlockingQueue<>(),
+                new SynchronousQueue<>(),
```

Comment:
in such case, you could set up `CachedThreadPoolExecutor` without this custom one.
there is `SynchronousQueue` and ttl 60sec inside that executor.